src/main/java/net/pms/formats/v2/AudioAttribute.java

```diff
@@ -29,11 +29,9 @@
  * boolean getLargerValue, Integer defaultValue, Integer minimumValue)
  */
 public enum AudioAttribute {
-	BITRATE (set("BitRate"), true, true, AudioProperties.BITRATE_DEFAULT, 1),
 	CHANNELS_NUMBER (set("Channel(s)"), true, true, 2, 1),
 	DELAY (set("Video_Delay"), false, false, 0, null),
-	SAMPLE_FREQUENCY (set("SamplingRate"), true, true, 48000, 1),
-	BITS_PERSAMPLE (set("BitDepth"), true, true, 16, 1);
+	SAMPLE_FREQUENCY (set("SamplingRate"), true, true, 48000, 1);
```

Comment:
The logic is to keep what is already there and as such can be used by older plugins - but not to add anything new.